src/snappy_device_agents/devices/multi/tests/test_multi.py

```diff
@@ -27,6 +29,14 @@ def submit_job(self, job_data):
         return str(uuid4())
 
 
+def test_bad_tfclient_url():
+    """Test that Multi raises an exception when TFClient URL is bad"""
+    with pytest.raises(ValueError):
+        _ = TFClient(None)
+    with pytest.raises(ValueError):
+        _ = TFClient("foo.com")
```

Comment:
I guess you can remove assignments here at all
```suggestion
        TFClient("foo.com")
```

Reply:
Fixed, thanks! For some reason I thought pylint would complain if it wasn't actually assigned, but it seems ok with it.